### DISCUSSION TITLE
License verification keeps going wrong

### DISCUSSION THREAD

**Discussion by ignism** [payloadcms/payload#304] [Q&A] [answered]
After creating a droplet on digital ocean and setting up my first user through `/admin/` I keep running into the license is invalid error. I initially made a license for the wrong domain as I didn't include the subdomain. But now everything should be in order.
I double checked my `.env` variables.
the domain is `http://payload.grndctrl.io`, if that helps.

What do?

**Comment by denolfe** [comment, marked as answer]
Hey @ignism, it looks like you had some canceled licenses with the same domain that were likely interfering. I've cleared out all of those, give it another shot when you can 👍

**Reply by ignism** [reply]
Thanks, it works now! I was afraid I messed up there. Would be nice if we had a chance in the future to clean that up ourselves.
Can't wait to give this puppy a spin.
